### ISSUE TITLE
Suggestion: Allowing wallpapers to extend to top & bottom toolbars

### ISSUE THREAD

**Issue by silverwings15** [lovac42/DancingBaloney#6]
Hiya, just wondering if it's feasible to have both the toolbars be covered by the wallpaper for overall congruence. i've seen some people using a workaround of chopping an image into 3 parts and then reattaching them, but that method isn't possible with an animated background.

an example of the inconguence i mention:
![img](https://i.imgur.com/aYKnFTa.png)

**Comment by lovac42** [comment]
Makes sense. Hiding the top causes issues, I don't want to deal with that. But hidding the bottom seems fine.

Next version:
![image](https://user-images.githubusercontent.com/42665127/77718337-07e06200-6fb9-11ea-8762-be20e2303aa1.png)

![image](https://user-images.githubusercontent.com/42665127/77718082-5e996c00-6fb8-11ea-94df-7f0b9e27b4a9.png)

**Comment by silverwings15** [comment]
amazing, thank you very much. if possible, can you elaborate on what issues would arise from making the top toolbar transparent?
